src/resources/app-fr.json

```diff
@@ -1021,6 +1020,8 @@
   "noResponseIdentifier": "Aucun identifiant de réponse pour le moment.",
   "viewingResponsesSubmitted": "Consultation des réponses soumises le :",
   "responseIdentifierWithColon": "Identifiant de réponse :",
+  "respondentIdentifierWithColon": "Intimé :",
```

Comment:
I believe this might have gotten mistranslated by Google Translate – we have been using the word `Répondant` for respondent in the French language file.